src/NovelRT.DotNet/CMakeLists.txt

```diff
@@ -115,22 +104,24 @@ elseif(UNIX)
   set(nethost_lib "libnethost.so")
 endif()
 
-add_library(CoreCLR::nethost SHARED IMPORTED GLOBAL)
-set_target_properties(CoreCLR::nethost
+add_library(Dotnet SHARED IMPORTED GLOBAL)
+add_dependencies(Dotnet Dotnet_Build)
+set_target_properties(Dotnet
   PROPERTIES
   INTERFACE_INCLUDE_DIRECTORIES "${CMAKE_CURRENT_BINARY_DIR}"
   IMPORTED_LOCATION "${CMAKE_CURRENT_BINARY_DIR}/${nethost_lib}"
```

Comment:
Since we're now using the `RESOURCE` property to share additional build outputs, we can set the primary output file here, which is the .NET host library which is needed to link against, and also communicate the additional files necessary (in this case, the .NET wrapper libraries)